2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Configure stats pid to 2579599
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Loading settings from /home/sahanp/.config/wandb/settings
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Loading settings from /home/sahanp/triton-data/wandb/settings
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-13 16:25:33,507 WARNING MainThread:2579599 [wandb_setup.py:_flush():79] Could not save program above cwd: /home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/recipes/full_finetune_single_device.py
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/recipes/full_finetune_single_device.py', 'program': '/home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/recipes/full_finetune_single_device.py'}
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_init.py:_log_setup():533] Logging user logs to /home/sahanp/triton-data/wandb/run-20241113_162533-3u8ff0mc/logs/debug.log
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_init.py:_log_setup():534] Logging internal logs to /home/sahanp/triton-data/wandb/run-20241113_162533-3u8ff0mc/logs/debug-internal.log
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_init.py:init():619] calling init triggers
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_init.py:init():669] starting backend
2024-11-13 16:25:33,507 INFO    MainThread:2579599 [wandb_init.py:init():673] sending inform_init request
2024-11-13 16:25:33,508 INFO    MainThread:2579599 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-13 16:25:33,508 INFO    MainThread:2579599 [wandb_init.py:init():686] backend started and connected
2024-11-13 16:25:33,509 INFO    MainThread:2579599 [wandb_init.py:init():781] updated telemetry
2024-11-13 16:25:33,520 INFO    MainThread:2579599 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-11-13 16:25:34,170 WARNING MainThread:2579599 [wandb_init.py:init():1260] interrupted
Traceback (most recent call last):
  File "/home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 817, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 279, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sahanp/.conda/envs/parity-bench/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 122, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sahanp/.conda/envs/parity-bench/lib/python3.11/threading.py", line 629, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sahanp/.conda/envs/parity-bench/lib/python3.11/threading.py", line 331, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-11-13 16:25:34,173 WARNING MsgRouterThr:2579599 [router.py:message_loop():75] message_loop has been closed
